src/icons/README.md

```diff
@@ -0,0 +1,64 @@
+# Custom Icons
+
+This directory exports custom icons from the `src/assets/icons/` folder.
+
+**Important:** Only SVG files placed in `src/assets/icons/` are considered icons. Logos and other images should remain in `src/assets/img/`.
+
+## Adding New Icons
+
+Thanks to `vite-plugin-svgr`, you can easily add new SVG icons without creating wrapper components each time.
+
+### Steps to Add a New Icon:
+
+1. **Add your SVG file** to `src/assets/icons/` (not `src/assets/img/`)
+   - Use snake_case for filenames (e.g., `my_icon.svg`)
+   - Ensure the SVG has `viewBox="0 0 24 24"` for consistency
+   - Use `fill="currentColor"` for colorable icons
+
+2. **Add export to `src/icons/index.tsx`**
+
+   ```typescript
+   // Add to the "Custom icons from assets/icons/" section
+   export const IconMyIcon = createIconComponent(
+     require('../assets/icons/my_icon.svg?react').default
+   );
+   ```
+
+   Or simply import at the top and add the export:
+
+   ```typescript
+   import MyIconSvg from '../assets/icons/my_icon.svg?react';
+   // ... then in exports section:
+   export const IconMyIcon = createIconComponent(MyIconSvg);
```

Comment:
One thing to note here is that since the exports are re-exported together with the tabler icons, it's possible to overwrite a tabler icon without realizing. To address this we might want to add a custom suffix to our icons such as `IconAdministratorCustom`. If you have any other idea, let me know. Maybe some static check is also possible.